### PR TITLE
feat: [SRM-15363]: adding SRM module to new Side Nav framework

### PR DESCRIPTION
**Pull request by harshilbits**
### Summary
1. Adding SRM module to new Side Nav framework

#### Screenshots
New Note
New Note
New Note

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by deepesh-ui — APPROVED** [review]

**Review by praneshg239 — APPROVED** [review]

File: src/modules/85-cv/CVRouteDestinations.tsx [hunk 1]
```diff
@@ -32,24 +33,41 @@ import CVHomePage from './pages/home/CVHomePage'
...
+  }
+
+  if (scope === Scope.ACCOUNT) {
     return (
       <Redirect
         to={routes.toCVSLOs({
-          projectIdentifier: identifier,
-          orgIdentifier,
+          projectIdentifier: params?.projectIdentifier,
+          orgIdentifier: params?.orgIdentifier,
```
Comment (SahilKr24) [inline src/modules/85-cv/CVRouteDestinations.tsx:62]: just a thought, if this is a account scope path, why are we passing project and org to it? 🤔

Comment (harshilbits) [reply src/modules/85-cv/CVRouteDestinations.tsx:62]: updated.

File: src/modules/10-common/RouteDefinitionsV2.ts [hunk 2]
```diff
@@ -157,6 +159,121 @@ const routes = {
   // cf routes
   toFeatureFlags: withModeModuleAndScopePrefix<ModulePathParams>(() => '/feature-flags'),
 
+  // cv routes
+  toMonitoredServices: withModeModuleAndScopePrefix(() => '/monitoredservices'),
+  toAddMonitoredServices: withModeModuleAndScopePrefix(() => '/monitoringservices/setup'),
+  toCVDashboard: withModeModuleAndScopePrefix(() => `/cv`),
```
Comment (avinashmadhwani02) [inline src/modules/10-common/RouteDefinitionsV2.ts:165]: /cv is not required. `withModeModuleAndScopePrefix` already supports module name

File: src/modules/10-common/RouteDefinitionsV2.ts [hunk 3]
```diff
@@ -157,6 +159,121 @@ const routes = {
...
+    return `monitoringservicesinputset`
+  }),
+  toAccountCVSLODetailsPage: withModeModuleAndScopePrefix<{ identifier: string }>(
+    params => `/${module}/slos/${params?.identifier}`
+  ),
+  toCVSLOs: withModeModuleAndScopePrefix(() => {
+    return `slos`
+  }),
+  toAccountCVSLOs: withModeModuleAndScopePrefix(() => {
+    return `/cv/slos`
```
Comment (avinashmadhwani02) [inline src/modules/10-common/RouteDefinitionsV2.ts:210]: same here

File: src/modules/85-cv/CVRouteDestinations.tsx (hunk 1 above)
Comment (avinashmadhwani02) [reply src/modules/85-cv/CVRouteDestinations.tsx]: +1
we can remove project and org

File: src/modules/10-common/RouteDefinitionsV2.ts (hunk 2 above)
Comment (harshilbits) [reply src/modules/10-common/RouteDefinitionsV2.ts:165]: okay

**Review by avinashmadhwani02 — APPROVED** [review]

File: src/modules/10-common/RouteDefinitionsV2.ts (hunk 3 above)
Comment (harshilbits) [reply src/modules/10-common/RouteDefinitionsV2.ts:210]: done.

File: src/modules/10-common/pages/SettingsPages/ProjectSettingsPage.tsx [hunk 4]
```diff
@@ -154,6 +154,12 @@ export const ProjectSettingsPage: React.FC = () => {
               icon={'variables-blue'}
               route={routesV2.toVariablesSettings({ accountId, orgIdentifier, projectIdentifier, module })}
             />
+            <SettingsResourceCard
+              label={<String stringID="common.sloDowntimeLabel" />}
+              id={SettingsResources.SLODownTime}
+              icon={'connectors-blue'}
+              route={routesV2.toCVSLODowntime({ accountId, orgIdentifier, projectIdentifier, module })}
+            />
```
[the remark this answers is not in the document]
Comment (harshilbits) [reply src/modules/10-common/pages/SettingsPages/ProjectSettingsPage.tsx:162]: Hi @Sagar-Rajput7 , this will be done by Code Error team.

**Comment by harshilbits** [comment]
Type check is failing in the feature branch itself and not part of new changes in this PR.

File: src/modules/10-common/pages/SettingsPages/ProjectSettingsPage.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (SagarRajput7) [reply src/modules/10-common/pages/SettingsPages/ProjectSettingsPage.tsx:162]: got it

**Pull request merged**